docker-compose.yml

```diff
@@ -0,0 +1,47 @@
+version: "3.7"
+services:
+  curity:
+    container_name: idsvr
+    environment:
+      PASSWORD: Secr3t
+    image: curity/idsvr
+    ports:
+      - "6749:6749"
+      - "4466:4466"
+    networks:
+      demonetwork:
+        aliases:
+          - idsvr
```

Comment:
Do we need this **alias** when there is a **container_name**?

Reply:
yes, the alias specifies which hostname the container will have in the specified network thus makes it possible to refer to that name in the configurations. The hostname is not the same as the container_name since the default hostname is the containerid: https://docs.docker.com/config/containers/container-networking/#ip-address-and-hostname

It may be more clear if we specify the hostname explicitly instead of an alias.